clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/WebConfig.groovy

```diff
@@ -48,17 +55,24 @@ import javax.servlet.Filter
 @Configuration
 @Import([
   SimpleCORSFilter,
+  ApplicationsController,
```

Comment:
I suspect scanning `com.netflix.spinnaker.clouddriver.controllers` is going to be a less cumbersome path forward than explicit @Imports.